logrus/example_logrus_test.go

```diff
@@ -0,0 +1,54 @@
+// Licensed to SkyAPM org under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. SkyAPM org licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package logrus
+
+import (
+	"context"
+	"os"
+
+	"github.com/SkyAPM/go2sky"
+	"github.com/sirupsen/logrus"
+)
+
+func ExampleWrapFormat() {
+	context := context.Background()
+
+	logrus.SetOutput(os.Stdout)
+
+	// init tracer
+	_, err := go2sky.NewTracer("example")
+	if err != nil {
+		logrus.Fatalf("create tracer error %v \n", err)
+	}
+
+	// json format
+	logrus.SetFormatter(Wrap(&logrus.JSONFormatter{
+		DisableTimestamp: true,
+	}, "SW_CTX"))
+	logrus.WithContext(context).Info("test1")
+
+	// test format
+	logrus.SetFormatter(Wrap(&logrus.TextFormatter{
+		DisableTimestamp: true,
+	}, "SW_CTX"))
+	logrus.WithContext(context).Info("test2")
+
+	// Output:
+	// {"SW_CTX":"[,,N/A,N/A,-1]","level":"info","msg":"test1"}
+	// level=info msg=test2 SW_CTX="[,,N/A,N/A,-1]"
```

Comment:
Let's add the output into the doc too.

Reply:
But more like real data.

Reply:
> Let's add the output into the doc too.

already add the context format at https://github.com/SkyAPM/go2sky-plugins/pull/19/files#diff-f583953a040e7174c73614f1e1e0d8e81695a399c931598d2e28a4b2bb5494cfR22.

Reply:
> But more like real data.

Golang will check output is the same with stdout, test failure if not the same. Should I need to add some real data to the comment?